test/Microsoft.Extensions.Logging.Testing.Tests/AssemblyTestLogTests.cs

```diff
@@ -51,70 +53,124 @@ public void TestLogWritesToITestOutputHelper()
         }
 
         [Fact]
-        public async Task TestLogWritesToGlobalLogFile()
+        public Task TestLogWritesToGlobalLogFile() =>
+            RunTestLogFunctionalTest((tempDir, loggerFactory) =>
```

Comment:
Not sure I like this pattern of using an expression body and then putting the real thing in a lambda.

Reply:
Testing out @pakrym's suggestion from https://github.com/aspnet/KestrelHttpServer/pull/2433#issuecomment-376751160

Reply:
@anurse but you can put everything on one line this way :)